gendo.py

```diff
@@ -3,6 +3,7 @@
 import os
```

Comment:
Also you should `import calendar` from the standard library to use day of week constants. See below